### ISSUE TITLE
Re-remove [role="_abstract"]

### ISSUE THREAD

**Issue by rolfedh** [redhat-documentation/modular-docs#221]
**TLDR**

- Somehow the changes in https://github.com/redhat-documentation/modular-docs/pull/185 were reversed. 
- Despite some sleuthing, I couldn't figure out how that happened.
- Doing it again.

**Context**
Reposted from a slack thread:

ifrangs
  [8 hours ago](https://redhat-internal.slack.com/archives/C04QRCA35K8/p1706699878664349)
I have just checked and the mod docs templates for the [concept](https://github.com/redhat-documentation/modular-docs/blob/main/modular-docs-manual/files/TEMPLATE_CONCEPT_concept-explanation.adoc?plain=1), [procedure](https://github.com/redhat-documentation/modular-docs/blob/main/modular-docs-manual/files/TEMPLATE_PROCEDURE_doing-one-procedure.adoc?plain=1), and [reference](https://github.com/redhat-documentation/modular-docs/blob/main/modular-docs-manual/files/TEMPLATE_REFERENCE_reference-material.adoc?plain=1) topics still contain the [role="_abstract"] tag, which according to the mod docs guide were removed in April 2022:
1 reply

rdlugyhe
  [12 minutes ago](https://redhat-internal.slack.com/archives/C04QRCA35K8/p1706726273908559?thread_ts=1706699878.664349&cid=C04QRCA35K8)
It's quite strange. If you look at [the blame for the templates](https://github.com/redhat-documentation/modular-docs/blame/main/modular-docs-manual/files/TEMPLATE_ASSEMBLY_a-collection-of-modules.adoc), you see that [role="_abstract"] was added 4 years ago by [issue-126 updated the annotated template files](https://github.com/redhat-documentation/modular-docs/commit/80af607a9c1180ab01d09e41c69db8c92e1b9ef7) and [remain there today](https://github.com/redhat-documentation/modular-docs/blob/main/modular-docs-manual/files/TEMPLATE_ASSEMBLY_a-collection-of-modules.adoc?plain=1) even though [issue-184 Removed the abstract tag from all templates](https://github.com/redhat-documentation/modular-docs/pull/185) (PR [185](https://github.com/redhat-documentation/modular-docs/pull/185)) removed them about 2 years ago. (edited)
